media/css/firefox/family/components/modules/_privacy.scss

```diff
@@ -19,4 +19,39 @@
             background-color: f3.$green-light;
         }
     }
+
+    @media #{$mq-sm} {
+        .l-grid {
+            display: grid;
+
+            .c-subtitle {
+                min-width: 8ch;
+                max-width: 12ch;
+            }
+        }
+    }
+
+    @media #{$mq-lg} {
+        .l-grid {
+            grid-template-columns: minmax(20ch, 35ch) 1fr;
```

Comment:
**issue**: it looks like we're meant to leave more space on the browser column side
<img width="654" alt="Screen Shot 2022-08-31 at 1 17 18 PM" src="https://user-images.githubusercontent.com/19650432/187740272-8c090e96-97fc-4683-b014-f7716c2c4bb7.png">

**suggestion**: let's change the minmax to have a percentage max, i.e. 40%, then we should have space to put the blurb in column 2, under the browser